src/views/order_view.js

```diff
@@ -0,0 +1,55 @@
+import Backbone from 'backbone';
+import Quote from '../models/quote';
+// import QuoteView from '../views/quote_view';
+import Order from '../models/order';
+import OrderList from '../collections/order_list';
+import $ from 'jquery';
+
+
+const OrderView = Backbone.View.extend({
+  initialize(params) { //where do the params come from? app.js
+    this.template = params.template;
+    this.model = params.model;
+    this.listenTo(this.model, 'change', this.render);
+    //anytime the model changes, it will redraw it
+    this.listenTo(this.model.get('quote'), 'change', this.executeOrder);
+
+  },
+
+  executeOrder() {
+    console.log('execute order is being called');
+    let quote = this.model.get('quote');
```

Comment:
I think this might make more sense as a function defined on the `Order` model, since it only does data manipulation, and doesn't touch the DOM at all. To make sure the view is properly removed, you could add the following line to `initialize()` above:

```javascript
this.listenTo(this.model, 'destroy', this.remove);
```